.github/workflows/lint.yml

```
@@ -83,7 +83,7 @@ jobs:
 
   pr-sanity-checks:
     name: pr-sanity-checks
-    runs-on: [self-hosted, linux.large]
+    runs-on: amz2023.linux.large
```

Why remove `self-hosted` here?
```suggestion
    runs-on: [safe-hosted, amz2023.linux.large]
```

---

Cleanup, the extra label doesn't do anything since the only `linux.large` runner is our self hosted one. The `self-hosted` label was intended to handle the case when github offers a runner with the same label as a self hosted one.